Scroller/EditableWrapper.js

```diff
@@ -37,6 +40,12 @@ const EditableShape = PropTypes.shape({
 
 const SpotlightAccelerator = new Accelerator([5, 4]);
 
+const pressDuration = 500;
+
+const holdConfig = {
+	events: [{time: pressDuration}]
+};
```

Comment:
```suggestion
const holdConfig = {
	events: [
		{name: 'hold', time: pressDuration}
	]
};
```

Reply:
@jeonghee27 is right. I missed that `name` becomes `undefined` if we don't specify in `holdConfig`.

Reply:
It's fixed.